.yarnrc.yml

```diff
@@ -1,3 +1,5 @@
+nodeLinker: node-modules
```

Comment:
whyd you do this?

Reply:
this was automatically added iirc

Reply:
im guessing you may just be having some merge conflicts because i pushed to the repository changes in how yarn v2 was set up (this made the CI work). maybe try pulling in main and just reverting the changes to some of these config files if you didn't mean to make them